scripts/ristiinvertailu/Makefile

```diff
@@ -0,0 +1,86 @@
+SHELL := /bin/bash
+DATE_NOW := $(shell date '+%Y-%m-%d')
+ALKUPVM := 2024-11-14 # TODO: Change this!
+TILAT = 'odottaa_kasittelya','kysely_muodostettu','vastaajatunnus_muodostettu'
+
+# POSTGRES
+DB_NAME := ehoks
+DB_USERNAME := app
+DB_HOST := localhost
+DB_PORT := 55432
+
+ifdef AWS_PROFILE
+  ifeq ($(AWS_PROFILE), oph-prod)
+    ENV_NAME := sade
+  else
+    ENV_NAME := pallero
+  endif
+  DB_PASSWORD != aws ssm get-parameter --profile $(AWS_PROFILE) \
+  	--name /$(ENV_NAME)/postgresqls/ehoks/app-user-password \
+  	--with-decryption | jq '.Parameter.Value'
+endif
+
+.PHONY: clean plot
+
+all: plot
+
+outputs:
+	mkdir -p outputs
```

Comment:
Tää taitaa tehdä nyt sen, että kaikki targetit ajetaan uudestaan?

Reply:
Näin taitaa olla, sorry